README.md

```diff
@@ -64,15 +64,15 @@ and it is a simple app that will be used to perform mathematical calculations.
 - **[html]**
 - **[css]**
 - **[REACT]**
-- **[]**
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo<a name="live-demo"></a>
 
-- [Live Demo Link]()
+- [N/A]()
```

Comment:
- [ ] This line still represents a link that is broken and the previous reviewer suggested to write `N/A` as normal text. it translates into a link because the way we write down links in `markdown` language syntax.
what you can do write now is to remove line `75` and then paste this `-  N/A` on line 75.